src/constants/orderbook.ts

```diff
@@ -9,7 +9,7 @@ export const ORDERBOOK_ANIMATION_DURATION = 100;
  * @note ORDERBOOK_ROW_HEIGHT should be a divisor of ORDERBOOK_HEIGHT so that we do not have a partial row at the bottom
  * @note To change the Orderbook width, --orderbook-trades-width and ORDERBOOK_WIDTH must be changed to the same value
  */
-export const ORDERBOOK_HEIGHT = 800;
+export const ORDERBOOK_HEIGHT = 792;
 export const ORDERBOOK_WIDTH = 300;
-export const ORDERBOOK_ROW_HEIGHT = 24;
+export const ORDERBOOK_ROW_HEIGHT = 22;
```

Comment:
light thought, have we tried the values `21/756` here instead? I'm a fan of 4 point grid systems (aka having everything divisible by 4) visually